test/xframe.js

```diff
@@ -44,4 +44,120 @@ describe('XFRAME', function () {
             .expect(200, done);
     });
 
-});
\ No newline at end of file
+    it('header (sameorigin) on allowlist', function (done) {
+        var config = { xframe: {value: 'SAMEORIGIN', allowlist: ['/']} },
+            app = mock(config);
+
+        app.get('/', function (req, res) {
+            res.status(200).end();
+        });
+
+        request(app)
+            .get('/')
+            .expect('X-FRAME-OPTIONS', config.xframe.value)
+            .expect(200, done);
+    });
+
+    it('header (sameorigin) not on allowlist', function (done) {
+        var config = { xframe: {value: 'SAMEORIGIN', allowlist: ['/allow']} },
+            app = mock(config);
+
+        app.get('/', function (req, res) {
+            res.status(200).end();
+        });
+
+        app.get('/allow', function (req, res) {
+            res.status(200).end();
+        });
+
+        request(app)
+            .get('/allow')
+            .expect(200)
+            .end(function (err, res) {
+                var isHeaderPresent = res.header['x-frame-options'] !== undefined;
+                assert(isHeaderPresent);
```

Comment:
any reason not to use the expect syntax similar to line 57?

Reply:
good point, will change